test/sync/src/commonMain/kotlin/io/realm/test/mongodb/util/AdminApi.kt

```diff
@@ -72,12 +72,21 @@ interface AdminApi {
     suspend fun deleteAllUsers()
 
     /**
-     * Pause or re-enable Sync on the server. This will not cause existing sessions to fail,
+     * Deactivates Sync on the server. This will not cause existing sessions to fail,
      * they will instead attempt to reconnect later.
      */
     suspend fun pauseSync()
+
+    /**
+     * Activates Sync on the server.
+     */
     suspend fun startSync()
 
+    /**
+     * Terminates sync on the server and reactivate it again. It would trigger a client reset.
```

Comment:
But that isn't what happens, right? Instead we delete the client server files?

Reply:
Yes, I think this is leftovers from the first approach we implemented. The documentation needs to be updated.